README.md

```diff
@@ -1,10 +1,10 @@
-Team Foundation Server plugin for Jenkins
+Azure DevOps and Team Foundation Server plugin for Jenkins
 ============================
 Copyright &copy; Erik Ramfelt, Olivier Dagenais, CloudBees, Inc. and others.
 Licensed under [MIT Licence].
  
 ## Summary
-This plugin integrates [Team Foundation Version Control] (also known as TFVC) and Git to Jenkins by connecting to Team Foundation Server (TFS) and Visual Studio Team Services (Team Services). It also lets you trigger a release in Visual Studio Team Services, through a post-build step in Jenkins.
+This plugin triggers a release in Azure DevOps, through a post-build step in Jenkins.  It also integrates [Team Foundation Version Control] (also known as TFVC) and Git to Jenkins by connecting to Team Foundation Server (TFS). 
```

Comment:
TFS is rebranded to "AzureDevops Service" (VSTS) and "AzureDevops Server" (onprem)

I would replace "Team Foundation Server (TFS)" with "Azure Devops Service/Server"

Reply:
I am ok with leaving TFS in the docs now since many customers might still be on older versions and using TFS, not Azure DevOps Server.